lib/login/view/login_view.dart

```diff
@@ -1,35 +1,63 @@
-import 'package:auth_example/signup/view/signup_view.dart';
+import 'package:auth_example/login/bloc/login_bloc.dart';
+import 'package:auth_example/signup/view/signup_page.dart';
 import 'package:auth_example/home/view/home_view.dart';
-import 'package:auth_service/auth.dart';
-import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 
 class LoginView extends StatelessWidget {
   final TextEditingController _emailController = TextEditingController();
```

Comment:
maybe don't use controllers inside stateless and try the other approach we've discussed